Makefile

```diff
@@ -0,0 +1,93 @@
+RUSTUP = rustup
+
+TOOLCHAIN = $(shell cat rust-toolchain)
+CARGO = $(RUSTUP) run --install $(TOOLCHAIN) cargo --color always
+
+NIGHTLY_TOOLCHAIN = "nightly-2019-07-31"
+CARGO_NIGHTLY = $(RUSTUP) run --install $(NIGHTLY_TOOLCHAIN) cargo --color always
+
+GIT_HOOKS_PATH = ".githooks"
+GIT_HOOKS = $(wildcard $(GIT_HOOKS_PATH)/*)
+
+BUILD_ARGS = ""
+
+default: init_git_hooks build format
+
+init_git_hooks: $(GIT_HOOKS)
+	git config core.hooksPath $(GIT_HOOKS_PATH)
+
+install_rust:
+	$(RUSTUP) install $(TOOLCHAIN)
+
+install_rust_nightly:
+	$(RUSTUP) install $(NIGHTLY_TOOLCHAIN)
+
+## Dev environment
+
+install_clippy: install_rust
+	$(RUSTUP) component list --installed --toolchain $(TOOLCHAIN) | grep -q clippy || $(RUSTUP) component add clippy --toolchain $(TOOLCHAIN)
+
+# need nightly toolchain to get access to `merge_imports`
```

Comment:
I think we use more than `merge_imports` on fmt nightly.. no? I'd just remove this comment.

Reply:
True!